storage/storage.go

```diff
@@ -30,10 +30,11 @@ type User struct {
 
 // DB : The database struct to store stuff from the server
 type DB struct {
-	User   User
-	Item   Item
-	Number int
-	db     *gorm.DB
+	User    User
+	Item    Item
+	Number  int
```

Comment:
What's Number field for?

Reply:
Something I had in there to track the number of requests, not relevant tho